configuration/config-txt/boot.md

```diff
@@ -39,7 +39,7 @@ Set the `disable_commandline_tags` command to `1` to stop `start.elf` from filli
 
 ## arm_64bit
 
-If set to non-zero, forces the kernel loading system to assume a 64-bit kernel, starts the processors up in 64-bit mode, and sets `kernel8.img` to be the kernel image loaded, unless there is an explicit `kernel` option defined in which case that is used instead. Defaults to 0 on all platforms. **NOTE**: 64-bit kernels must be uncompressed image files.
+If set to non-zero, forces the kernel loading system to assume a 64-bit kernel, starts the processors up in 64-bit mode, and sets `kernel8.img` to be the kernel image loaded, unless there is an explicit `kernel` option defined in which case that is used instead. Defaults to 0 on all platforms. **NOTE**: 64-bit kernels may be uncompressed image files or a gzip archive of an image (which can still be called kernel8.img; the bootloader will recognize the archive from the signature bytes at the beginning).
```

Comment:
Is this now the same logic as the 32-bit kernel? (If it is, then perhaps this behaviour isn't worth pointing out specifically for the 64-bit kernel? :shrug: )

Reply:
No it's not the same. ARM32 kernel decompression (or not) is handled by the kernel image itself, whereas on ARM64 the loader is expected to take care of it.

Reply:
Ahhh, thanks for clarifying.